backend/src/impl/analyses/significance_analysis.py

```diff
@@ -0,0 +1,120 @@
+import numpy as np
+from explainaboard.info import SysOutputInfo
+from explainaboard.metrics.metric import MetricStats
+from explainaboard.metrics.registry import get_metric_config_class
+from explainaboard.utils.typing_utils import unwrap
+
+
+def update_metric_config(system_output: SysOutputInfo) -> SysOutputInfo:
+    metric_configs = [
+        get_metric_config_class(metric_config_dict["cls_name"])(**metric_config_dict)
+        for metric_config_dict in system_output.metric_configs
+    ]
+    # update metric_configs in system_output
+    system_output.metric_configs = metric_configs
+
+    return system_output
+
+
+"""
```

Comment:
Could you move this docstring within the function and use a standard format for arguments such as:

```
:param n_samples: The number of boostrapped samples
:param prop_samples: The ratio of samples to take every time
```

This is more standard, and also will help later if we start using an automatic document generator such as readthedocs.